ansible/tasks/installOpenwhiskCatalog.yml

```diff
@@ -0,0 +1,22 @@
+---
+# This task will install the standard actions and packages available in openwhisk-catalog.
+
+- name: the openwhisk-catalog URL and the location to download
+  set_fact:
+    openwhisk_catalog_url: "https://github.com/openwhisk/openwhisk-catalog.git"
+    openwhisk_catalog_home: "{{ openwhisk_home }}/../openwhisk-catalog"
+
+- stat: path={{ openwhisk_catalog_home }}
+  register: openwhisk_catalog_path
+ 
+- name: download the openwhisk-catalog to the openwhisk-catalog location if necessary
+  git:
+    repo: "{{ openwhisk_catalog_url }}"
+    dest: "{{ openwhisk_catalog_home }}"
+    depth: 1
+  when: openwhisk_catalog_path.stat.isdir is not defined
+
+- name: install the catalog from openwhisk-catalog
+  local_action: shell "./installCatalog.sh" chdir="{{ openwhisk_catalog_home }}/packages"
```

Comment:
I can change the script installCatalog.sh in openwhisk-catalog to receive these params.